app/assets/stylesheets/application.scss

```diff
@@ -1,7 +1,7 @@
 $theme-colors: (
   // "primary":    #11afaf,
```

Comment:
why does the primary theme color is commented out?

Reply:
Because its not in use for now. Im going to erase it. Commit: 386e64f